src/manipulators/moveNewHandleTouch.js

```diff
@@ -3,19 +3,43 @@ import external from '../externalModules.js';
 import triggerEvent from '../util/triggerEvent.js';
 import { clipToBox } from '../util/clip.js';
 
-export default function (eventData, toolType, data, handle, doneMovingCallback, preventHandleOutsideImage) {
-  // Console.log('moveNewHandleTouch');
+/**
+ * Moves a new handle using touch events
+ * @public
+ * @method moveNewHandleTouch
+ * @memberof CornerstoneTools.Manipulators
+ *
+ * @param {*} eventData
```

Comment:
stars -> types?

Reply:
`*` means "Any Type"
There are a lot of ways the JS Doc can still be improved. My main efforts here is cleaning up eslint issues, adding namespaces, getting API doc generation complete, making it easier for others to contribute by including clear examples, and resolving the bulk of our public/private issues.

Still a decently long way to go until we've resolved all of the issues like this.